### PR TITLE
Final formatting and added KPI examples

### PR DESCRIPTION
**Pull request by srivasud**
1) Someone removed the formatting. 
2) Added KPI examples.

### CODE REVIEW

File: 06-process.Rmd [hunk 1]
```diff
@@ -64,17 +64,17 @@ The below are the important prerequisites of a successful data visualization pro
...
 
 + **Deciding which visual is best**
-One of the biggest challenges for business users is deciding which visual should be used to best represent the information. Tables, Linechart, Areachart, Barchart, Scatterplots, Piechart, Treemaps, Heatmaps, Natual Language Generation, Etc.
+One of the biggest challenges for business users is deciding which visual should be used to best represent the information. Tables, Linechart, Areachart, Barchart, Scatterplots, Piechart, Treemaps, Heatmaps, Natual Language Generation, Etc. Depending upon the industry and specific department business may want to measure success based on specific goals and targets. Few of the KPI examples that can help are listed as follows: Marketing Metrics, Sales Metrics, Saas Metrics and KPIs, Digital Marketing Metrics, Social Media Metrics, SEO KPIs, Financial Metrics and KPIs, Supply Chain Metrics, Healthcare Metrics, Call Center Metrics, Support Metrics, Retail Metrics and KPIs, Help Desk Metrics and KPIs, Insurance Metrics, E-Commerce Metrics and KPIs, HR Metrics, Devops Metrics and KPIs.
```
Comment (harizhisham) [inline 06-process.Rmd]: Great addition, but could you list out the metrics in a bullet point format? It would help readability tremendously.

**Review by harizhisham — CHANGES_REQUESTED** [review]
Please see comments in "Deciding which visual is best"

File: 06-process.Rmd (hunk 1 above)
Comment (srivasud) [reply 06-process.Rmd]: Hi @harizhisham , Agreed! That's a good suggestion. I included the bullet formatting now.

**Review by srivasud — COMMENTED** [review]
This includes bullet format as well.

**Review by Marvin510 — APPROVED** [review]
looks good

**Review by Marvin510 — APPROVED** [review]

**Review by harizhisham — APPROVED** [review]

**Comment by harizhisham** [comment]
looks good. will approve and merge

**Pull request merged**